dumpling-overview: Add info about how to install with TiUP (#5350)
pingcap/docs: dumpling-overview.md

When using Dumpling, you need to execute the export command on a running cluster. This document assumes that there is a TiDB instance on the `127.0.0.1:4000` host and that this TiDB instance has a root user without a password.

> You can get dumpling with [tiup](/tiup/tiup-overview.md) by running `tiup install dumpling`. Afterwards you can run it as `tiup dumpling ...`.

Reviewer: ```suggestion
You can get Dumpling using [TiUP](/tiup/tiup-overview.md) by running `tiup install dumpling`. Afterwards, you can use `tiup dumpling ...` to run Dumpling.
```
